src/Models/Individuals/SLatissima.jl

```diff
@@ -118,15 +116,15 @@ Keyword Arguments
 
 - `architecture`: the architecture to adapt arrays to
 - `growth_rate_adjustement`, ..., `exudation_redfield_ratio`: parameter values
-- `pescribed_velocity`, `pescribed_temperature` and `pescribed_salinity`: functions for the relative velocity, temperature and salinity in the form `f(x, y, z, t)`
+- `prescribed_velocity`: functions for the relative velocity, temperature and salinity in the form `f(x, y, z, t)`
```

Comment:
Oh yeah I didn't update the docstring correctly
```suggestion
- `prescribed_velocity`: functions for the relative velocity `f(x, y, z, t)`
```

I do think this is something we would want to get rid of at some point but I think it might still be essential in some cases